vignettes/extending-jmpost.Rmd

```diff
@@ -232,3 +232,188 @@ parameter { real mu;      // non-whitespace after opening `{`
   x ~ normal(mu, sigma);
 } // some comment         // non-whitespace after closing `}`
 ```
+
+## Stan Data Objects
+
+When writing your own Stan code to extend jmpost it is important to note that
+many different data objects have already been defined in the `data` block of the
+base Stan template. This section outlines the different data objects that are
+made available to user. Note that some objects are only made available if
+the corresponding model is used; for example death times are only available if
+the user specifies a `SurvivalModel` object to `JointModel()`.
+
+### Global Data Objects
+
+**Number of unique subjects**
+```stan
+int<lower=1> Nind;
```

Comment:
proposal: if we say consistently "subjects", and if other such variables use snake_case and write out the words, I would think
```suggestion
int<lower=1> n_subjects;
```
would be more natural here (could find/replace globally)